test/DynamoCoreWpfTests/ViewExtensions/GraphNodeManagerViewExtensionTests.cs

```diff
@@ -191,8 +191,9 @@ public void FilterFrozenItemsTest()
 
         /// <summary>
         /// Test if the number of Nodes containing Null or Empty List matches what is shown on the UI
+        /// Marked as Failure until we can fix flakyness
         /// </summary>
-        [Test]
+        [Test, Category("Failure")]
```

Comment:
I saw a regression at [DynamoCoreWpfTests.GraphNodeManagerViewExtensionTests.CorrectNumberNodeItemsTest](https://master-15.jenkins.autodesk.com/job/DYN-Dynamo/job/DynamoBuildscripts/job/master/2630/testReport/junit/DynamoCoreWpfTests/GraphNodeManagerViewExtensionTests/CorrectNumberNodeItemsTest/). Is this failure category added to the wrong test?